src/examples/eet/eet-file.c

```diff
@@ -75,7 +83,15 @@ main(void)
    if (!create_eet_file())
      return -1;
 
-   ef = eet_open("/tmp/my_file.eet", EET_FILE_MODE_READ);
+   char *tmpdir = getenv("TEMP");
+   const char filename[] = "my_file.eet";
+   size_t tmpdir_len = strlen(tmpdir) + 1;
+   size_t filename_len = strlen(filename) + 1;
+   size_t file_len = tmpdir_len + filename_len;
+   char file[file_len];
+   snprintf(file, file_len, "%s" EINA_PATH_SEP_S "%s", tmpdir, filename);
```

Comment:
Maybe you'll prefer `eina_<something>_join_paths`? See if it has a significant difference.

Reply:
Like this?
```suggestion
   eina_file_path_join(file, file_len, tmpdir, tmpdir_len, filename, filename_len);
```

Reply:
> 
> 
> Like this?

Yes, somethingl ike that

Reply:
Is this way ok to you?
```diff
-   size_t filename_len = strlen(filename) + 1;
-   size_t file_len = tmpdir_len + filename_len;
-   char file[file_len];
-   snprintf(file, file_len, "%s" EINA_PATH_SEP_S "%s", tmpdir, filename);
+  char file[PATH_MAX];
+  eina_file_path_join(file, sizeof(file), tmpdir, filename);
```

Reply:
Using `PATH_MAX`, hm, nice catch. Seems to be pretty fine for me, indeed.

Reply:
You got my thumbs up 👍